net/tls.toit

```diff
@@ -0,0 +1,69 @@
+/**
+Tls example code.
+
+This example sends a request to Google.com and prints the size of the response.
+*/
+
+import net
+import net.x509 as net
+import http
+import tls
+
+main:
+  network_interface := net.open
+
+  host := "www.google.com"
+  tcp := network_interface.tcp_connect host 443
+
+  socket := tls.Socket.client tcp
+    --server_name=host
+    --root_certificates=[GLOBALSIGN_ROOT]
+
+  connection := http.Connection socket host
+  request := connection.new_request "GET" "/"
+  response := request.send
+
+  bytes := 0
+  while data := response.read:
+    bytes += data.size
```

Comment:
I think our example should do more.
Now I would need to look up another example to figure out how to actually deal with the data.
If you know how to easily handle the data, it should be trivial to change it to something more useful.
If you don't then it shows that we are missing something here... ;)
```suggestion
    // Discard the data from the server, and just count the number of bytes.
    bytes += data.size
```